docs/notebooks/gis.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
<p>Quelques suggestions:</p><p> </p><ul><li>The dataset needs a time dimension<code>, usually at a daily frequency with no missing timesteps (NaNs are supported). If your data differs from that, you'll need to be extra careful on the results provided.</code></li><li><code>While this is not necessary for get_yearly_op, variable names should be one of those supported <a href="https://github.com/Ouranosinc/xclim/blob/master/xclim/data/variables.yml" target="_blank">here</a> for maximum compatibility.</code></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>